spec/article_json/export/apple_news/elements/image_spec.rb

```diff
@@ -13,29 +13,43 @@
   let(:caption) { [ArticleJSON::Elements::Text.new(content: 'Foo Bar')] }
   let(:alt) { 'Text description' }
 
-  let(:expected_json) { {URL: '/foo/bar.jpg', caption: 'Foo Bar', role: 'image'} }
-  
+  let(:expected_json) do
+    [
+      {
+        role: 'image',
+        URL: '/foo/bar.jpg',
+      },
+      {
+        role: 'caption',
+        text: 'Foo Bar',
+        layout: 'captionLayout',
+        textStyle: 'captionStyle',
+      }
+    ].join(',')
```

Comment:
This won't produce JSON, but a string of concatenated `Hash.to_s`...

Reply:
Ditto.